### PR TITLE
Reformat PO files after translation tooling update

### PR DESCRIPTION
**Pull request by mattias-p**
I should have done this together with #573 but apparently I forgot.

* Formatting is updated.
* Obsolete messages are removed from all PO files.
* New translations are added for English only - the new message strings are exact copies of their respective message ids.
* The PO files for the other languages still lack translations for new messages.

### CODE REVIEW

**Comment by mattias-p** [comment]
@matsduf wrote:
> (There are 29 message tags in the en.po file that is missing in the other po files. I expecte those to be added even if no translation is available.)

How so?

**Comment by mattias-p** [comment]
I pushed a new commit that makes sure the comments get picked up by xgettext and inserted into the PO files.

The surface area of the commit is pretty big but it only consists of:
* A documentation tweak
* Source formatting changes according to the documentation tweak
* Updated line numbers because of the formatting changes
* The four extracted comments mentioned above

File: docs/Translation.pod [hunk 1]
```diff
@@ -19,17 +19,18 @@ by other software.
 Each test module must also have a method named C<tag_descriptions()>. This method
 must return a reference to a hash whose entries are expected to look like this:
 
-    FRIENDLY_GREETING => sub {    # FRIENDLY_GREETING
-        __x "Hello, {name}!", @_;
-    },
+    FRIENDLY_GREETING => sub {
+        __x    # FRIENDLY_GREETING
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply docs/Translation.pod]: Please @matsduf, would you approve the PR? This is not part of the substance of the PR. And nobody will misunderstand this.

[the remark this answers is not in the document]
Comment (matsduf) [reply docs/Translation.pod]: @mattias-p, hat is the problem of changing to make the text clearer?

[the remark this answers is not in the document]
Comment (matsduf) [reply docs/Translation.pod]: I will make the change in a separate PR instead and accept that it is not change in this PR.

**Pull request merged**